### DISCUSSION TITLE
Provide a general summary of the issue

### DISCUSSION THREAD

**Discussion by Henry3223** [twbs/bootstrap#35539] [Q&A]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Hi! I'm started in HTML around 1 month, I need your help! And sorry, the English is not my first language.

My div container and his rows seem to not  close properly. If I use the container class all items that come after seem to be inclued with the div, like as my footer. I have tried to find out the reason but I'm not able. I put border at all items to see exactly where the div closes and his rows and I did same with the background color.

Here is my complete code:

<!DOCTYPE html>
<html>
    <style>div{border: 2px solid;}</style>
    <head>
        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">

    </head>

    <body>
        <header>
            <nav class="navbar navbar-expand-lg navbar-light bg-dark sticky-bottom ">
                <div class="container-fluid">
                  <a class="navbar-brand text-primary" href="#"><img src="/docs/5.1/assets/brand/bootstrap-logo.svg" alt="" width="30" height="24" class="d-inline-block align-text-top">
                    Bootstrap</a>
                  <button class="navbar-toggler bg-primary" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                    <span class="navbar-toggler-icon"></span>
                  </button>
                  <div class="collapse navbar-collapse" id="navbarSupportedContent">
                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
                      <li class="nav-item">
                        <a class="nav-link active text-white" aria-current="page" href="#">Home</a>
                      </li>
                      <li class="nav-item">                       
                  </div>
                </div>
              </nav>
        </header>

        <div class=" container" >
               

            <div class="row bg-primary ">
          
                <div class="col-sm-3 ">
                      
                    <ol class="list-styled">
                        <ol>Liste d'items technologiques.</ol>
                        <li>Souris.</li>
                        <li>Écran.</li>
                        <li>Clavier.</li>
                        <li>Mémoires.</li>
                    <ol>
                </div>
           
                <div class="col-sm-8 ms-auto ">
                    <div  class = "row" >
                        <div class="col-md-6">
                            <p >Petite photo "funny"!</p>
                        </div>
                        <div class="col-sm-6">
                            <img src="./IMAGES/dot-com-1480211.jpg" alt="Photo *com">
                        </div>
                    </div>                    
            </div>
            <div class="row bg-white">
          
                <div class="col-12 col-sm-6">
                    <div class="card">
                        <img src=./IMAGES/teletext-8-1200253.jpg class="img-fluid">
                        <div class="card-body">
                            <h5 class="card-title">Hacking!</h5>
                            <p class="card-text">Le mode sombre du piratage informatique.</p>
                            <a href="#" class="btn btn-primary">Par ici!</a>
                        </div>
                    </div>
                </div>
                <div class="col-12 col-sm-6">
                    <div class="card">
                        <img src=./IMAGES/sony-playstation-portable-1536533.jpg class="img-fluid">
                        <div class="card-body">
                            <h5 class="card-title">Sony PSP</h5>
                            <p class="card-text">Petite console portable de Sony.</p>
                            <a href="#" class="btn btn-primary">Spécifications ici!</a>
                        </div>
                    </div>  
                </div>
            </div>
            <div class="row bg-danger">          
                <div class="col-12 col-sm-6">
                    <div class="card">
                        <img src=./IMAGES/teletext-8-1200253.jpg class="img-fluid">
                        <div class="card-body">
                            <h5 class="card-title">Hacking!</h5>
                            <p class="card-text">Le mode sombre du piratage informatique.</p>
                            <a href="#" class="btn btn-primary">Par ici!</a>
                        </div>
                  </div>
                </div>
                <div class="col-12 col-sm-6">
                    <div class="card">
                        <img src=./IMAGES/sony-playstation-portable-1536533.jpg class="img-fluid">
                        <div class="card-body">
                            <h5 class="card-title">Sony PSP</h5>
                            <p class="card-text">Petite console portable de Sony.</p>
                            <a href="#" class="btn btn-primary">Spécifications ici!</a>
                        </div>
                    </div>
                </div> 
            </div>
        </div>
          
         
          
        <footer class ="text-center bg-dark  text-primary">
                    <div >
                      <hr>
                      <p>   &copy;   Technique informatique – Automne 2021  </p>
                    </div>
                  
                  
        </footer>       
   
    </body>

</html>

.

### Reduced test cases

Sorry, I don't understand what you want here exactly.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5.1.3

**Comment by ffoodd** [comment]
You have an unclosed `<li class="nav-item">` in your nav. That may explain your issue.

Please check your HTML using the [W3C markup validation](https://validator.w3.org/).

For your information, checking the source code of your page using `ctrl`+`u` in your browser should highlight such errors in red.

Does it help?
